src/components/pages/InstructorHome/index.css

```diff
@@ -1,14 +1,12 @@
 .site-calendar-demo-card {
-  width: 50%;
+  width: 100%;
```

Comment:
Let's make sure we convert this to a LESS file tomorrow so we can have some consistency across the project.